### PR TITLE
Code cleanup, refactor, envvars

### PR DESCRIPTION
**Pull request by Index01**
- Frontend cleanup complete.

### CODE REVIEW

**Review by otakup0pe — COMMENTED** [review]
I know js but don't really know react but nothing here seems too spooky?

**Comment by Index01** [comment]
@otakup0pe nothing too spooky confirmed. 
The react really just says, "fire off these async requests for me" as part of their components. We have an App.js, which has Pages, which have Components and lots of conditions for populating those components with data. 

Biggest change in the frontend refactor is eliminating the Login components and making it part of the Login page. Seemed simpler as that component will only ever be a part of the login page. 

More to come in the backend refactor.

File: backend/populate_reservations.py [hunk 1]
```diff
@@ -200,6 +200,7 @@ def create_guests(init_file="", init_rooms=""):
     characters = string.ascii_letters + string.digits + string.punctuation
 
     for guest_new in dr:
+        print(f'new guest: {guest_new}')
```
Comment (mpesavento) [inline backend/populate_reservations.py:203]: Eventually we will want to have some stable logging

File: backend/reservations/views/login.py [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+        print(f'data: {data}')
+        try:
+            guests = Guest.objects.all()
+            guest_email = guests.filter(email=data['email'])
+        except KeyError as e:
+            logger.info(f"[-] User login failed {data['email']}")
+            return Response("User not found", status=status.HTTP_400_BAD_REQUEST)
+        logger.info(f"[+] User login attempt: {data['email']}")
+        print(f"[+] User login attempt: {data['email']}")
+        #TODO(tb): FixMe
```
Comment (mpesavento) [inline backend/reservations/views/login.py:43]: This looks fixed to me? If JWT is working, let's remove this comment

**Review by mpesavento — APPROVED** [review]
Looks great! Thanks for cleaning this up!

File: backend/populate_reservations.py (hunk 1 above)
Comment (Index01) [reply backend/populate_reservations.py:202]: Ja all the `/views/` have python logging setup. 
If we decide to keep using this script adding logging would be good but if we move most the functionality to a /admin page prolly best to wait(?)

**Pull request merged**